package.json

```diff
@@ -8,6 +8,7 @@
     "react-dom": ">=15.3.0"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.4.0",
```

Comment:
We can remove reselect as createSelector is exported from @redux/toolkit and hence we don't need reselect explicitly.